View/Helper/VideoHelper.php

```diff
@@ -77,7 +77,8 @@ public function youtube($url, $settings = array()) {
 			'autoplay' => 0,
 			'loop' => 0,
 			'enablejsapi' => 0,
-			'showinfo' => 1
+			'showinfo' => 1,
+			'class' => ''
```

Comment:
@samdharris could you update the code so that it only shows the class attribute if a class is set? I believe you need to set the `class` to `null` by default. It will need updating in a few places.

```suggestion
			'class' => null
```